### ISSUE TITLE
Displaying 3 revision requests - 1 exists

### ISSUE THREAD

**Issue by S-Haime** [openlibhums/janeway#4522]
**Describe the bug**
Three revision requests are showing, despite only 1 existing. (See[ FD #9463](https://janeway.freshdesk.com/a/tickets/9463) )
![image](https://github.com/user-attachments/assets/800de17a-5d9d-4b99-afdc-024d203edfec)
The user did indicate they went 'back and forth through the process' a few times:

"I started the request yesterday, and had it open overnight. During that time, I was wrestling with the text editor box and might also have gone a few times back and forth through the first and second steps (Editor's Note and email respectively). I finalized and sent the email today, and noticed the revision is listed three times, but the logs show only one email sent. "

There is only 1 revision request in JW (checked through admin), but three are displayed in the back office.

**Janeway version**
1.73

**To Reproduce**
Start revision request, go back and forth throughout the process around midnight.

**Comment by S-Haime** [comment]
Other example: https://janeway.freshdesk.com/a/tickets/9656

**Comment by ajrbyers** [comment]
I don’t think this is a bug. Three assignments were created if they went back and forward.  More likely a UX/UI consideration

**Comment by S-Haime** [comment]
@ajrbyers I think it's the display part - if there were 3 requests in admin as well it would make sense to me. But with there being 1 in admin and 3 displayed, it veers on bug territory. But definitely also a UI consideration (and maybe one for @Astronaut-resting to find out *why* they're going back and forth)

**Comment by ajrbyers** [comment]
Ah there looks like there is a bug with the Edit in Admin link. I'll update this ticket.

**Comment by ajrbyers** [comment]
Backlog refinement: If a user attempts to open a revision request whilst there is still underway we should use the confirmation form to check that they really do want to open a new one. A link to edit/view the existing one should be displayed.

**Comment by S-Haime** [comment]
Seems to have been a freak accident, has not been reported since or reproduced; there have been changes to this page and template since as well.
